tests/test_auth_api.py

```
@@ -0,0 +1,137 @@
+        ).status_code
+        == 200
+    )
+
+
+def test_change_password_wrong_current(client, login):
+    login(client)
+    resp = client.post(
+        "/api/auth/password",
+        json={"current_password": "bad", "new_password": "newpw"},
+    )
+    # 当前密码错误是校验失败（400），不是会话过期（401）
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)